dialogue-core/src/main/java/com/palantir/dialogue/core/TracedChannel.java

```diff
@@ -58,7 +58,7 @@ private static ImmutableMap<String, String> tracingTags(Config cf, Endpoint endp
         return ImmutableMap.<String, String>builderWithExpectedSize(endpointTags.size() + configTags.size())
                 .putAll(endpointTags)
                 .putAll(configTags)
-                .build();
+                .buildOrThrow();
```

Comment:
we may want this to be `buildKeepingLast()`

Reply:
I think failure indicates a bug in DialogueTracing factory methods, throw is probably reasonable (less confusion if we clobber tags, anyhow)